lib/command.js

```diff
@@ -194,6 +200,11 @@ module.exports = function (yargs, usage, validation) {
       if (maybe.variadic) argv[maybe.cmd] = argv._.splice(0)
       else argv[maybe.cmd] = argv._.shift()
       postProcessPositional(yargs, argv, maybe.cmd)
```

Comment:
we should also apply `postProcessPositional()` to both the normal form, and expanded form of the argument; perhaps we could make `postProcessPositional()` take an array? and `addCamelcaseExpansions ()` could return an array of either: the original argument, or the original argument plus the camel-case expansion?

We'd need to add one more test, verifying that `coerce` is applied to the camel-case version of the key (added along-side the existing coerce test).